### PR TITLE
Switch from CoreCompare.System.Drawing to System.Drawing

### PR DESCRIPTION
**Pull request by rootyElf**
Replacing CoreCompat.System.Drawing with System.Drawing.Common. This helps resolving issues with native libraries on some Linux distributions. (Arch Linux for instance does not provide libpcre.so.3 which the CoreCompat.System.Drawing runtime apparently needs)
Natives for CoreCompat.System.Drawing are also only available for x86 (if I'm not mistaken).

### CODE REVIEW

**Comment by peppy** [comment]
Thanks for the contribution. Could you please let us know which operating system/architectures you've tested this on?

**Comment by rootyElf** [comment]
Hi, I'm actually that guy: https://www.reddit.com/r/SwitchHacks/comments/8osp7e/osulazer_on_nintendo_switch_gameplay_linux/e09thfb
 
It works on my X64 Arch Linux machine using dotnet core. As I said, without this change it wants me to have libpcre.so.3 which is not provided by arch by default. This could maybe be worked around by doing some symlinking, but I think this might be a cleaner solution.

**Comment by rootyElf** [comment]
And, yeah it still crashes on the Switch and I currently don't have another ARM machine I could test it on.

**Comment by peppy** [comment]
Got it. This was our planned direction, anyways. Appreciate further testing by others to ensure we don't regress on any platforms.

**Comment by rootyElf** [comment]
Sure

File: osu.Framework/osu.Framework.csproj [hunk 1]
```diff
@@ -21,10 +21,8 @@
     <PackageTags>osu game framework</PackageTags>
   </PropertyGroup>
   <ItemGroup Label="Package References">
-    <PackageReference Include="CoreCompat.System.Drawing.v2" Version="5.2.0-preview1-r131" />
+    <PackageReference Include="System.Drawing.Common" Version="4.5.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="2.7.0" />
-    <PackageReference Include="runtime.linux-x64.CoreCompat.System.Drawing" Version="1.0.0-beta009" />
-    <PackageReference Include="runtime.osx.10.10-x64.CoreCompat.System.Drawing" Version="5.4.0-r8" />
```
Comment (smoogipoo) [inline osu.Framework/osu.Framework.csproj]: Looks like we need to include this for macOS, `mono-libgdiplus` isn't installed by default. Need to check if the same is true for linux.

Comment (Tom94) [reply osu.Framework/osu.Framework.csproj]: Same holds true for linux (and the respective linux runtime).

**Pull request merged**